package.json

```diff
@@ -1,89 +1,89 @@
 {
-  "private": true,
```

Comment:
Could you undo the indentation changes in this file?

Reply:
Yep, don't know how that got in there.